cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -133,14 +138,22 @@
                 </span>
                 {% endif %}
                 <div class="{{ _classes( nav_depth, '-lists' ) }}">
-                    {{ _nav_list( nav_depth, nav_items ) }}
+                    {{ _nav_list( nav_depth, nav_groups ) }}
                 </div>
+               {% if footer %}
+          <div class="nav_footer">
+            {% for item in footer %}
+            <p>{{ item | safe }}</p>
+            {% endfor %}
+          </div>
+         {% endif %}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3147/commits/194f5f2c31c5342a26fe78c7342602083a7681a4